.travis.yml

```diff
@@ -5,8 +5,8 @@ os:
   - linux
   # - osx
 julia:
-  - 1.0
-  - 1.1
+  - 1.3
+  - 1.5
```

Comment:
why not 1?

Reply:
AutomotiveVisualization.jl has a [compat] statement with Julia v1.3 and v1.4 (shown here: https://github.com/sisl/AutomotiveVisualization.jl/blob/master/Project.toml#L25)

And the Travis build threw an error because of this: https://travis-ci.org/github/sisl/AutoUrban.jl/builds/772105890

Reply:
ok but I meant if we put 1, it should always use the latest julia (i.e. 1.6)
Anyway we can merge it now :)

Reply:
Oh I did not know that! Good to know :) Thanks, Maxime